jekyll/_cci2/api-permissions.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: classic-docs
+title: "API Permissions"
+short-title: "API Permissions"
+description: "How to Add and Change API Permissions on CircleCI"
+order: 20
+---
+
+To use the CircleCI API
+or view details about a build,
+you will need API tokens with the appropriate permissions.
+This document describes the types of API tokens,
+as well as how to create and delete them.
+
+## Overview
+
+There are two types of API token
+you can create within CircleCI.
+
+  1. **Personal**:
+  These tokens are used to interact with the CircleCI API
+  and grant full read and write permissions.
+  2. **Project**:
+  These tokens provide information about specific projects
+  and only grant read permissions.
```

Comment:
You can't use Project tokens to trigger?

Reply:
Hmm. If that's the case, which scope would it need to be set at? The doc on [triggering jobs conditionally](https://circleci.com/docs/2.0/api-job-trigger/#conditionally-running-jobs-with-the-api) only says you need the "API token from your project's settings page".